### PR TITLE
Enable optee in mixins

### PR DESCRIPTION
**Pull request by jingdlu**

### CODE REVIEW

File: groups/device-specific/celadon_ivi/framework_manifest.xml [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
+        <name>android.hardware.keymaster</name>
+	<version>3.0</version>
+	<transport>hwbinder</transport>
+        <interface>
+            <name>IKeymasterDevice</name>
+            <instance>default</instance>
+        </interface>
+    </hal>
+    <!--
     <hal format="aidl">
```
Comment (syan10) [inline groups/device-specific/celadon_ivi/framework_manifest.xml:17]: suggest to split this into 2 commits, one is for enable optee basic and xtest, the other is for keymaster related changes

File: groups/optee/true/init.rc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    chown system system /persist
+    chmod 0771 /persist
+    restorecon /persist
+    mkdir /persist/tee 0770 shell shell
+    restorecon_recursive /persist/tee
+
+    start tee-supplicantd
+
+    service tee-supplicantd /vendor/bin/tee-supplicant
```
Comment (JeevakaPrabu) [inline groups/optee/true/init.rc:13]: no need to indent this line

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]
Call out all the changes done in detailed commit message

File: groups/optee/false/product.mk [hunk 3]
```diff
@@ -0,0 +1,3 @@
+PRODUCT_PACKAGES += \
+    android.hardware.gatekeeper@1.0-service.software \
```
Comment (syan10) [inline groups/optee/false/product.mk:2]: remove this

File: groups/optee/true/BoardConfig.mk [hunk 4]
```diff
@@ -0,0 +1,12 @@
+OPTEE_PLATFORM ?= kbl
```
Comment (syan10) [inline groups/optee/true/BoardConfig.mk:1]: it should not only target to kbl, right?

Comment (jingdlu) [reply groups/optee/true/BoardConfig.mk:1]: https://github.com/op-tee-ia/optee_os only supports kbl platform. This is just for TA compiling, not for real TEE running.

File: groups/optee/false/product.mk (hunk 3 above)
Comment (jingdlu) [reply groups/optee/false/product.mk:2]: Will remove this file for now.

File: groups/optee/true/init.rc (hunk 2 above)
Comment (jingdlu) [reply groups/optee/true/init.rc:13]: Thanks for your comments. BTW, I will remove kernel driver config change for this time, will commit it with optee driver patch together.

File: groups/device-specific/celadon_ivi/framework_manifest.xml (hunk 1 above)
Comment (jingdlu) [reply groups/device-specific/celadon_ivi/framework_manifest.xml]: Will remove keymaster change for this time.

**Review by JeevakaPrabu — APPROVED** [review]

**Pull request merged**